v1/components/MarkdownEditor.tsx

```diff
@@ -30,11 +28,27 @@ const MarkdownEditor = () => {
 		observer.observe(mdeSizeElm)
 
 		return () => observer.disconnect()
-	}, [isVisible])
+	}, [])
 
 	return (
 		<div ref={containerRef} className="min-h-[400px] bg-white">
-			<SimpleMDE id="simple-mde" value={markdownValues} onChange={onChange} />
+			<SimpleMDE
+				id="simple-mde"
+				value={markdownValues}
+				onChange={onChange}
+				onFocus={() => {
+					if (!field) {
+						return
+					}
+					handleFieldFocusEvent({ eventName: FOCUS_EVENTS.FOCUS })

```

Comment:
```suggestion
 field && handleFieldFocusEvent({ eventName: FOCUS_EVENTS.FOCUS })
```

same suggestion for below